lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/metrics/TDigestHistogram.java

```diff
@@ -53,17 +49,28 @@ public static TDigestHistogram fromValue(double value, double weight)
     }
 
     @JsonCreator
+    public TDigestHistogram(@JsonProperty("digest") String digest)
```

Comment:
A `byte[]` a would be a more natural representation for the serialized form. Jackson should be able to deal with them natively by converting them to a suitable string representation.

Reply:
Yeah. Makes sense. I've also extracted a @JsonCreator to a static method annotated with @DoNotCall to make it explicit that it's just for jackson.